armips/move/battle_sub_seq/358.s

```diff
@@ -14,7 +14,6 @@
 a001_358:
     changevar2 VAR_OP_SET, VAR_HP_TEMP, VAR_HIT_DAMAGE
     if IF_EQUAL, VAR_HP_TEMP, 0x0, _0030
-    // removed the command to half the value
 _0030:
```

Comment:
to be clear, when this if passes it still just jumps direct to `_0030`--no need to have the if